proposals/0320-codingkeyrepresentable.md

```diff
@@ -20,7 +20,7 @@ Swift-evolution thread: [[Pitch] Allow coding of non-`String`/`Int` keyed `Dicti
 
 The primary motivation for this pitch lays in the much-discussed confusion of this default behavior:
 
-* [Dictionarys encoding strategy](https://forums.swift.org/t/dictionarys-encoding-strategy/11973)
+* [Dictionaries encoding strategy](https://forums.swift.org/t/dictionarys-encoding-strategy/11973)
```

Comment:
This _is_ misspelled, but it is quoting a thread title which is itself misspelled, so it is appropriate to leave it alone.